document_splitter/get_clusters.py

```diff
@@ -23,13 +23,14 @@ class ClusterDetection:
     def __init__(self, *args, **kwargs) -> None:
         super(ClusterDetection, self).__init__(*args, **kwargs)
 
-    def get_clustered_data(self, documents: List, data_distinction_type=DataType) -> pd.DataFrame:
+    def get_clustered_data(self, documents: List, data_distinction_type=DataType, n_samples=8) -> pd.DataFrame:
```

Comment:
Please be consistent. If type-hinting, hint for all arguments.

Reply:
Link - https://github.com/SalesHero-Inc/hero-document-clustering/pull/2/files#diff-13019bd0f86e8857233988bd38d768d259149464c9301444f46a556369f54a2fR29